benchopt/datasets/tests/test_dataset_simulated.py

```diff
@@ -1,5 +1,6 @@
 import pytest
 import numpy as np
+from scipy import sparse
```

Comment:
Use nested import to avoid paying the price of import

Reply:
can you explain more ?

Reply:
:+1: